### PR TITLE
Mentee appointment modal

### PR DESCRIPTION
**Pull request by Lam7150**
## Description :star2:

Screen Shot 2020-12-04 at 3 38 43 PM
Screen Shot 2020-12-04 at 3 38 57 PM
Screen Shot 2020-12-04 at 3 39 05 PM

Fixes #43

### CODE REVIEW

File: frontend/src/components/css/MenteeModal.scss [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+    flex-direction: row;
+    align-items: center;
+}
+
+
+.modal-mentee-availability-switch-text {
+    justify-content: center;
+    font-size: 14px;
+    font-weight: 700;
+    color: "#7A7A7A";
```
Comment (chyku) [inline frontend/src/components/css/MenteeModal.scss:109]: add colors, consts, to _app.scss

**Review by chyku — CHANGES_REQUESTED** [review]

File: frontend/src/components/MenteeAppointmentModal.js [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+  GENDERS,
+  ETHNICITIES,
+  AGES,
+} from "../utils/consts";
+import "./css/AntDesign.scss";
+import "./css/Modal.scss";
+import "./css/MenteeModal.scss";
+
+// TODO: Temporary constants, fill in later
+const organizations = ["Hack4Impact", "YMCA"];
```
Comment (chyku) [inline frontend/src/components/MenteeAppointmentModal.js:18]: wait do you need this? isn't organizations a text box

File: frontend/src/components/pages/Videos.js [hunk 3]
```diff
@@ -1,7 +1,5 @@
 import React from "react";
 
-function Videos() {
-  return <div>Videos</div>;
-}
+function Videos() {}
```
Comment (chyku) [inline frontend/src/components/pages/Videos.js:3]: why lol isn't this gonna cause merge conflicts

**Review by chyku — APPROVED** [review]
lgtm, few questions abt things

**Review by Feeshay — APPROVED** [review]
whooooo

The style consts inside the components feelsbadman but antd be that way

**Pull request merged**